webui/src/lib/components/repository/tree.jsx

```diff
@@ -648,10 +648,7 @@ const GetStarted = ({ config, onUpload, onImport }) => {
           <Button
             variant="link"
             className="mb-1"
-            disabled={
-              config.config.blockstore_type === "local" ||
-              config.config.blockstore_type === "mem"
-            }
+            disabled={config.config.blockstore_type === "mem"}
```

Comment:
Suggestion: Add a configuration flag to local and allow only if explicitly enabled